features/support/env.rb

```diff
@@ -73,7 +73,10 @@ def initialize_environment
 
 After do
   if @last_run_result && !@error_expected
-    expect(@last_run_result.error).to be_falsy, 'Expected no runtime error'
+    if @last_run_result.error
+      puts unformatted_last_run_output
+      expect(@last_run_result.error).to be_falsy, 'Expected no runtime error'
+    end
```

Comment:
Hmm the weird part to me is in the block we know the expectation will fail. This seems a bit weird to me. What do you think about:
```ruby
puts unformatted_last_run_output if @last_run_result.error
expect(@last_run_result.error).to be_falsy, 'Expected no runtime error'
```